packages/wdio-repl/src/index.ts

```diff
@@ -52,15 +52,15 @@ export default class WDIORepl {
     private _isCommandRunning = false
     private _replServer: repl.REPLServer | undefined
 
-    constructor(config: ReplConfig) {
+    constructor(config?: ReplConfig) {
         this._config = Object.assign(
             DEFAULT_CONFIG,
             { eval: this.eval.bind(this) },
             config
         )
     }
 
-    eval (cmd: string, context: vm.Context, filename: string, callback: (err: Error | null, result: any) => void) {
+    eval (cmd: string, context: vm.Context, filename: string | undefined, callback: (err: Error | null, result: any) => void) {
```

Comment:
I think this didn't work because it complaint that there can only be optional parameter at the end of the parameter chain. Given that `callback` is required here I had to put `string | undefined` instead of using the optional notation.